src/pretix/control/forms/event.py

```diff
@@ -240,6 +240,21 @@ class EventSettingsForm(SettingsForm):
         help_text=_("Require customers to fill in the names of all attendees."),
         required=False
     )
+    attendee_emails_asked = forms.BooleanField(
+        label=_("Ask for attendee e-mails"),
+        help_text=_("Ask for an e-mail address for all tickets which include admission to the event. Important: For "
```

Comment:
I don't think this is very clear … maybe more of a positive wording?

```
Attendee emails addresses are required regardless of this setting for confirmation links. This setting adds the possiblity to ask for a second email address, eg for group orders.
```

… and/or rename the setting label - if you have to explain that you mean something different, it might be better to rename. `Ask for secodary email address` or something like that?